src/concrete/ml/onnx/convert.py

```diff
@@ -167,6 +168,11 @@ def get_equivalent_numpy_forward_from_onnx(
             forward.
         check_model (bool): set to True to run the onnx checker on the model.
             Defaults to True.
+        lsbs_to_remove (Optional[Tuple[int, int]]): Contains the values of the least significant
```

Comment:
if we are still going towards this approach (which I'm still against), I still think this docstring should insist more on the fact that this is not something to be used for other things thatn tree-based models. For now, we only mention the `during tree traversal only` part, which I find ambiguous without context. Also, I'm not sure why the following explains the tree's structure (from `The first value` to the end), it does not seem to belong here as this is a very generic function. In my opinion, the docstring should look like : 
```
This parameter is only used for optimizing tree-based models. It contains the values of the least significant bits to remove. See more information on rounding in the documentation. It should never be used for other types of models. Default to None, for other types of models.
```

this is only an example, you don't need to copy pasted that exactly

Reply:
I didn't say: Detailed information on rounding can be found in the

Because this lsbs_to_remove will be removed, when we will add truncate